package.json

```diff
@@ -47,7 +47,7 @@
     "lint:scss": "stylelint {demosplan/DemosPlanCoreBundle/Resources/client/scss,projects}/**/*.scss"
   },
   "dependencies": {
-    "@demos-europe/demosplan-ui": "^0",
+    "@demos-europe/demosplan-ui": "^0.1.12",
```

Comment:
I assume this should stay as is

Reply:
I changed this back to a non specific version here: https://github.com/demos-europe/demosplan-core/pull/1486/commits/2a273471cb6e6a28185ccf5e72f046d0b3972817